### ISSUE TITLE
SfSignaturePad: extremely bad performance in iOS after upgrading Flutter version

### ISSUE THREAD

**Issue by gabrielbertollo** [syncfusion/flutter-examples#808]
The SfSignaturePad widget started performing very badly on iOS devices, seemingly after upgrading from Flutter 3.7 to 3.10. It is almost unusable for me.

The problem can be reproduced by creating a sample app only with the SfSignaturePad widget (code below) and running on any iOS device or simulator.
```dart
import 'package:flutter/material.dart';
import 'package:syncfusion_flutter_signaturepad/signaturepad.dart';

void main() {
  runApp(const MainApp());
}

class MainApp extends StatelessWidget {
  const MainApp({super.key});

  @override
  Widget build(BuildContext context) {
    return const MaterialApp(
      home: Scaffold(
        body: SfSignaturePad(),
      ),
    );
  }
}
```

**Comment by LavanyaGowtham2021** [comment]
We have checked the reported issue with [SfSignaturePad](https://pub.dev/documentation/syncfusion_flutter_signaturepad/latest/signaturepad/SfSignaturePad-class.html) and have reproduced it at our end. This issue is replicated with 3.10.0 and later Flutter SDK versions because Flutter uses an impeller rendering engine for iOS by default. Disabling the impeller by following the steps mentioned in the below link can resolve your issue.


Link: https://docs.flutter.dev/perf/impeller#ios

**Comment by Yuvaraj-Gajaraj** [comment]
HI @gabrielbertollo ,

We would like to inform you that the reported issue 'Facing performance and rendering issues while drawing a signature in the SignaturePad widget, replicated in versions 3.10.0 and later,' has been resolved in the latest Flutter SDK version 3.22 from the Flutter framework itself. Kindly upgrade your Flutter SDK to version 3.22.
 
Please find the SDK version compatibility with our widgets below
https://help.syncfusion.com/flutter/system-requirements#sdk-version-compatibility
